apps/admin-gui-e2e/src/integration/group-management-perun_admin.spec.js

```diff
@@ -83,14 +89,16 @@ context('Actions', () => {
 
     cy.intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
       .get('[data-cy=attributes]')
-        .click()
+      .click()
       .get(`[data-cy=${dbGroupAttribute}-checkbox]`)
-        .click()
+      .click()
       .get('[data-cy=remove-attributes]')
-        .click()
+      .click()
       .get('[data-cy=delete-attributes]')
-        .click()
+      .click()
       .wait('@removeAttributes')
+      .intercept('**/attributesManager/getAttributes/g**').as('getAttributes')
+      .wait('@getAttributes')
```

Comment:
```suggestion
      .intercept('**/attributesManager/getAttributes/g**').as('getAttributes')
      .get('[data-cy=delete-attributes]')
      .click()
      .wait('@removeAttributes')
      .wait('@getAttributes')
```

The proposed change is not working because `removeAttributes` request will not fire unless the click. I propose that the two requests wait for each other. The test will wait for the request to finish by this suggested change.